pypeit/core/meta.py

```diff
@@ -153,6 +154,7 @@ def define_additional_meta(nlamps=20):
                        'pressure': dict(dtype=float, comment='Pressure (units.bar) at observation time'),
                        'seq_expno': dict(dtype=int, comment='Number of exposure in observing sequence'),
                        'slitwid': dict(dtype=float, comment='Slit width, sometimes distinct from decker'),
+                       'slitlength': dict(dtype=float, comment='Slit length, used only for long slits'),
```

Comment:
Why do we need slitlength, decker, and decker_basename? Can you explain

Reply:
They are necessary for the reduction of `LONGSLIT` masks. I added some comments in the code.